recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb

```diff
@@ -17,12 +17,9 @@ MAINTAINER = "chao.zeng@siemens.com"
 SRC_URI = " \
     git://github.com/node-red/node-red-nodes;protocol=https \
     file://0001-add-the-board-info-add-the-led-control-node.patch \
-    file://0002-extend-gpio-to-D19.patch \
-    file://0003-Clean-up-mraa-objects-on-node-closing.patch \
-    file://0004-Add-USER-button.patch \
-    file://0005-Add-configurable-GPIO-mode-for-DIN.patch \
-    file://0006-node-red-gpio-Add-support-for-initial-message-from.patch"
-SRCREV="3087e8e2a1ea189f394bca0a2af159ad859d7722"
+    file://0002-Clean-up-mraa-objects-on-node-closing.patch \
```

Comment:
I think we can submit this change upstream as well. The interface was added to mraa, wasn't it?

Reply:
Yes, we can, I'm planning to do it, just waiting for mraa to be pushed.